crates/wasmi/src/lib.rs

```diff
@@ -86,7 +86,7 @@
 #[macro_use]
 extern crate alloc;
 #[cfg(feature = "std")]
-extern crate std as alloc;
+extern crate alloc;
```

Comment:
This is just wrong. We want to use the `std` lib if the user specifies to do so.

Reply:
But no code actually use `std` alias. I consider it a code style but forget to hold on it.